src/features/vote/PastRequests.tsx

```diff
@@ -136,7 +138,11 @@ const PastRequests: FC<Props> = ({
                     <div>{el.vote}</div>
                   </td>
                   <td>
-                    <div>{el.reward}</div>
+                    <div>
+                      {el.vote === el.correct && el.reward === "N/A"
+                        ? "Rewards Claimable"
```

Comment:
why not change `el.reward` to say "Rewards Claimable" rather than "N/A"?  i think if thats the case you dont have to edit the view, though im not sure

Reply:
Lightest change as a lot of other stuff is looking for "-" or "N/A"